docs/myriad-configuration.md

```diff
@@ -23,6 +23,9 @@ frameworkFailoverTimeout: 43200000
 # Myriad's mesos framework name.
 frameworkName: MyriadAlpha
 
+# Myriad's mesos framework role.
+frameworkRole: someRoleName
```

Comment:
@smarella Thank you for good catch.  I'll do that not only for this but also others.